certified-connectors/Enadoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,108 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3",
+    "title": "Enadoc",
+    "description": "Enadoc is an innovative, cloud-based enterprise document imaging system.",
```

Comment:
@enadoc-dev The description looks very short, we recommend you to provide a detailed description of your connector. 
Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)

Reply:
@sriyen-msft Thanks, I have added some more details in the description.